package.json

```diff
@@ -139,7 +139,7 @@
     "@vue/cli-plugin-unit-jest": "5.0.8",
     "@vue/cli-plugin-vuex": "5.0.8",
     "@vue/cli-service": "5.0.8",
-    "@vue/eslint-config-standard": "6.1.0",
+    "@vue/eslint-config-standard": "8.0.1",
```

Comment:
It looks like that has a `@vue/eslint-config-standard-with-typescript` now (from their [releases list](https://github.com/vuejs/eslint-config-standard/releases/tag/%40vue%2Feslint-config-standard-with-typescript%408.0.0)) — should we switch to that instead?

Reply:
Hmm, new vue-cli projects (targeting vue 2) still use the combo of 

```
"@vue/eslint-config-standard": "^6.1.0",
"@vue/eslint-config-typescript": "^9.1.0",
```

Even better, modern day vue 3 projects come with even less out of the box

```

but I'm fine with changing if that means we will be able to simplify

Reply:
From digging deeper, both `@vue/eslint-config-standard` and `@vue/eslint-config-standard-with-typescript` seems to not have updated their dependencies in two years, and everything there is outdated.  So we shouldn't touch it in this PR, but look into using something else in the future.